README.md

```diff
@@ -208,27 +180,224 @@ schema = infer(headers, values)
 
 The number of rows used by `infer` can be limited with the `row_limit` argument.
 
-### CLI
+### Validate
+
+Given a schema as JSON file, url to JSON file, or a Python dict, `validate` returns `True` for a valid JSON Table Schema, or raises an exception, `SchemaValidationError`.
+
+```python
+import io
+import json
+
+from jsontableschema import validate
+
+filepath = 'schema_to_validate.json'
+
+with io.open(filepath) as stream:
+    schema = json.load(stream)
+
+is_valid = jsontableschema.validate(schema)
```

Comment:
With the current API `jsontableschema.validate` is only used to catch exceptions. What I mean is that `is_valid` will never be `False`. This method should always be inside a `try/catch` block, so I suggest changing this example to something like:

```python
try:
    jsontableschema.validate(schema)
except jsontableschema.exceptions.SchemaValidationError as e:
   # handle errors
```

For this reason, the return value of `jsontableschema.validate` doesn't matter. I wouldn't return anything to make this clearer.